src/Control/Monad/Logic/Sequence/Internal.hs

```diff
@@ -95,7 +99,7 @@ deriving instance Monad m => Functor (View m)
 -- it's really defined this way! However, the real implementation is different,
 -- so as to be more efficient in the face of deeply left-associated `<|>` or
 -- `mplus` applications.
-newtype SeqT m a = SeqT (Queue (m (View m a)))
+newtype SeqT m a = SeqT { runSeqT :: (Queue (m (View m a))) }
```

Comment:
I don't think I ended up using `runSeqT` in code, but I keep adding it because generally I feel like it's nice to have named projections like this.

Reply:
Wouldn't `unSeqT` be more idiomatic these days? I'm not sure there's much point adding a record selector like that for an internal type, but it's your call of course.